File: app/src/main/java/com/chenenyu/router/demo/TestActivity.java
package com.chenenyu.router.demo;

import android.os.Bundle;
import android.support.v7.app.AppCompatActivity;

import com.chenenyu.router.annotation.Route;

@Route("test")
public class TestActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_test);
    }
}

File: Sample/kotlin-app/src/main/java/com/chenenyu/router/kotlin_app/TestActivity.kt
package com.chenenyu.router.kotlin_app

import android.os.Bundle
import android.support.v7.app.AppCompatActivity
import com.chenenyu.router.annotation.Route

@Route("test")
class TestActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_test)
    }
}
